admin_manual/windmill_workflows/index.rst

```diff
@@ -0,0 +1,60 @@
+==================
+Windmill Workflows
+==================
+
+Nextcloud integrates the Windmill workflow engine (https://www.windmill.dev/) to allow advanced custom workflows interacting with your Nextcloud instance.
+
+Installation
+------------
+
+ * Install Windmill
+
+   * Either as a standalone install or via the Windmill External App in Nextcloud (see :ref:`External Apps<ai-app_api>`)
+
+ * Enable the ``webhook_listeners`` app that comes with Nextcloud
+
+.. code-block:: bash
+
+   occ app:enable webhook_listeners
+
+Selecting the right Workspace
+-----------------------------
+
+When installing Windmill as an ExApp make sure to select the right workspace on the first run: Only the pre-existing "nextcloud" workspace is hooked up to nextcloud's internal event system, all other workspaces will need manual webhook setups for each workflow.
+
+Building a workflow
+-------------------
+
+Each workflow in windmill is a listener to a Nextcloud Webhook Event. If you are using the ExApp-packaged windmill, it will automatically register webhooks for the workflows you build using the following mechanism. If you are not using the ExApp-packaged windmill install then you will have to register webhooks for your workflows manually via the webhook_listeners API: see https://docs.nextcloud.com/server/latest/developer_manual/_static/openapi.html#/operations/webhook_listeners-webhooks-index
```

Comment:
```suggestion
Each workflow in Windmill is a listener to a Nextcloud Webhook Event. If you are using the ExApp-packaged Windmill, it will automatically register webhooks for the workflows you build using the following mechanism. If you are not using the ExApp-packaged windmill install then you will have to register webhooks for your workflows manually via the webhook_listeners API: see https://docs.nextcloud.com/server/latest/developer_manual/_static/openapi.html#/operations/webhook_listeners-webhooks-index
```

Maybe the Windmill endpoint to run a workflow should be mentioned as what needs to be set as the webhook target URL in case of using an "independent" Windmill instance.